_sass/elements/_links.scss

```diff
@@ -7,18 +7,23 @@
 // 
 // Styleguide elements.links
 
-a,
-.link-alpha {
-  border: none;
-  color: $base-font-color;
+%link {
   cursor: pointer;
   text-decoration: none;
+  border: none;
   transition: color 0.1s linear;
+}
+
+
+a,
+.link-alpha {
+  @extend %link;
+  color: $base-font-color;
   &:active,
   &:hover {
     text-decoration: none;
-    border-bottom: 1px dotted $base-link-color;
-    img{
+    border-bottom: $thin-border-size dotted $base-link-color;
+    img, i, span{
```

Comment:
Rule declaration should be preceded by an empty line<br>Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space